src/client/theme-default/utils.ts

```diff
@@ -21,3 +21,22 @@ export function isActive(route: Route, path?: string): boolean {
 export function normalize(path: string): string {
   return decodeURI(path).replace(hashRE, '').replace(extRE, '')
 }
+
+/**
+ * get the path without filename (the last segment). for example, if the given
+ * path is `/guide/getting-started.html`, this method will return `/guide/`.
+ * Always with a trailing slash.
+ */
+export function resolvePath(path: string): string {
```

Comment:
I think `resolvePath` is a bit unclear here, would it better to be `getPathDirname` or something?

Reply:
Good point! Let me change this one

Reply:
Done 🙌